app/assets/javascripts/leaflet.map.js

```diff
@@ -44,6 +77,32 @@ L.OSM.Map = L.Map.extend({
       this.baseLayers.push(layer);
     }
 
+    if (L.OpenMapTiles) {
+      var copyright_link = $("<a>", {
+        href: "/copyright",
+        text: I18n.t("javascripts.map.openstreetmap_contributors")
+      }).prop("outerHTML");
+
+      var copyright = I18n.t("javascripts.map.copyright_text", { copyright_link: copyright_link });
+
+      var openmaptiles_link = I18n.t("javascripts.map.openmaptiles", {
+        openmaptiles_url: "https://openmaptiles.org/",
+        maptiler_url: "https://www.maptiler.com/"
+      });
+
+      var terms = $("<a>", {
+        href: "https://wiki.osmfoundation.org/wiki/Terms_of_Use",
+        text: I18n.t("javascripts.map.website_and_api_terms")
+      }).prop("outerHTML");
+
+      this.baseLayers.push(new L.OpenMapTiles({
+        attribution: copyright + ". " + openmaptiles_link + ". " + terms,
+        code: "V",
```

Comment:
How do we allocate the single-letter codes? "V" is nowhere in the name of this layer.

"O" was formerly used for OPNVKarte, not sure if we can reuse it.

Reply:
I think it’s just based on the name of the layer, using whatever letter is available and unlikely to clash with the others. OpenHistoricalMap’s fork has also assigned [its own codes](https://github.com/OpenHistoricalMap/ohm-website/blob/ad4ea829a7f794a456ef5c1484e977653cd9f7c3/app/assets/javascripts/leaflet.map.js#L81), currently J, O, R, and W, but there hasn’t been a conflict because some of OSM’s featured layers have been removed. The only risk from reusing a previously assigned letter is that some old links to the old layer will now point to the new layer, but these links have been falling back to the Standard layer anyways.